Type error when running the project

Hi,

maybe I did something wrong, but using the versions ...
```
npm --version = 10.8.2
node --version = v20.19.2
```
... I get the following type error.

```
npm run start

> gitlab-2-github@0.1.5 start
> node node_modules/ts-node/dist/bin.js ./src/index.ts

/home/<username>/test/node-gitlab-2-github/node_modules/ts-node/src/index.ts:750
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/index.ts:55:36 - error TS2345: Argument of type 'typeof throttling' is not assignable to parameter of type 'OctokitPlugin'.
  Types of parameters 'octokit' and 'octokit' are incompatible.

(...)
Property 'retryCount' is missing in type 'import("/home/<username>/test/node-gitlab-2-github/node_modules/@octokit/rest/node_modules/@octokit/request/node_modules/@octokit/types/dist-types/OctokitResponse").OctokitResponse<any, number>' but required in type 'import("/home/<username>/test/node-gitlab-2-github/node_modules/@octokit/request/node_modules/@octokit/types/dist-types/OctokitResponse").OctokitResponse<any, number>'.
```
(Plus some more output that does not seem that useful to me).

I've cloned a fresh copy of the repository today. 

Does someone have any Idea what I should do to fix this or to debug it further?

Hi, 

I'm using node v23.8.0

When I try to add
```

 commitMap: {
    [key: string]: string;
  };

```

under export default interface Settings {

It causes my other settings file go fully red with underlines and won't run 

`settings.ts:3:16 - error TS2352: Conversion of type '{ gitlab: { url: string; token: string; projectId: number; listArchivedProjects: true; sessionCookie: string; }; github: { owner: string; ownerIsOrg: true; token: string; token_owner: string; repo: string; recreateRepo: false; }; ... 16 more ...; mergeRequests: { ...; }; }' to type 'Settings' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'commitMap' is missing in type '{ gitlab: { url: string; token: string; projectId: number; listArchivedProjects: true; sessionCookie: string; }; github: { owner: string; ownerIsOrg: true; token: string; token_owner: string; repo: string; recreateRepo: false; }; ... 16 more ...; mergeRequests: { ...; }; }' but required in type 'Settings'.

  3 export default {
                   ~
  4   gitlab: {
    ~~~~~~~~~~~
... 
 68   },
    ~~~~
 69 } as Settings;
    ~~~~~~~~~~~~~

  src/settings.ts:9:3
    9   commitMap: {
        ~~~~~~~~~
    'commitMap' is declared here.
`

What am I doing wrong?

--- a/sample_settings.ts
+++ b/sample_settings.ts
@@ -32,6 +32,10 @@ export default {
     'gitlabgroup/projectname.1': 'GitHubOrg/projectname.1',
     'gitlabgroup/projectname.2': 'GitHubOrg/projectname.2',
   },
+  commitMap: {
+    'gitlab-commit-hash-1': 'github-commit-hash-1',
+    'gitlab-commit-hash-2': 'github-commit-hash-2',
+  },
   conversion: {
     useLowerCaseLabels: true,
     addIssueInformation: true,

--- a/src/settings.ts
+++ b/src/settings.ts
@@ -35,6 +35,9 @@ export default interface Settings {
     log: boolean;
   };
   s3?: S3Settings;
+  commitMap: {
+    [key: string]: string;
+  };
 }
 
 export interface GithubSettings {

